examples/realtime_ctlog_tasker.py

```diff
@@ -35,9 +35,13 @@ def print_callback(message, context):
                     logger.info("Sends POST to https://<YOUR-API-ENDPOINT>/dev/ondemand/portscan")
                     logger.info("Triggered a Port Scan of: {}".format(fqdn))
 
-                    # TODO: add relevant tasking call here to /ondemand/observatoryscan
-                    logger.info("Sends POST to https://<YOUR-API-ENDPOINT>/dev/ondemand/observatory")
-                    logger.info("Triggered an Observatory Scan of: {}".format(fqdn))
+                    # TODO: add relevant tasking call here to /ondemand/httpobservatoryscan
+                    logger.info("Sends POST to https://<YOUR-API-ENDPOINT>/dev/ondemand/httpobservatory")
```

Comment:
Wonder if we should use an environment variable here instead of a placeholder that should be manually replaced in the code?
I think if we use an environment variable it would be more modular, and in the documentation we'd just say: "To run the examples just set the XYZ environment variable to your Lamdba host."

Reply:
I see that as mostly as placeholder for generic tasking client logic.  The intent is not to leave it that way, but more defer that piece of work till later.

Reply:
No worries, works for me.